app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<% @contestants.each do |contestant| %>
+  <div id="contestant-<%= contestant.id %>">
+    <%= contestant.name %><br>
+    <p>Projects:
+    <% contestant.projects.each do |project| %>
```

Comment:
You could do nested iterating here like you have but you could also write a method in your `Contestant` model to get all of the projects names. Something like:
```ruby
def projects_list
  projects.pluck(:name).to_sentence
end
```
Then. you would just call that method on `contestant` instead of iterating over `projects`.